dist/button/ds6/button.css

```diff
@@ -19,8 +20,7 @@ a.cta-btn {
 }
 a.cta-btn {
   background-color: #ffffff;
-  border-style: solid;
-  border-width: 1px;
+  border: 1px solid;
```

Comment:
This `border: 1px solid;` already appears to be defined in group selector above.

Reply:
Ah, let me grab that.